app/src/androidTest/java/org/simple/clinic/TestClinicApp.kt

```diff
@@ -96,7 +97,7 @@ class TestClinicApp : ClinicApp() {
             }
           }
 
-          override fun providePatientConfig(): Single<PatientConfig> {
+          override fun providePatientConfig(): Observable<PatientConfig> {
```

Comment:
Interesting, why did you convert this to an infinite stream?

Reply:
1. When we do remote config, they'll have to be `Observable`.
2. We realised that we were manually converting the `Single<PatientConfig>` to an observable wherever we were using it.

We decided to make this provide an `Observable` directly based on these two factors.

Reply:
makes sense